blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -708,7 +708,7 @@ public Boolean apply(Row<Locator, Long> row) {
         return map;
     }
 
-    public Set<Locator> getEnumLocatorsFromLocatorSet(Set<Locator> locators) throws Exception{
+    public HashSet<Locator> getEnumLocatorsFromLocatorSet(Set<Locator> locators) throws Exception{
         HashSet<Locator> locatorHashSet = new HashSet<Locator>();
```

Comment:
NIT: This change isn't necessary is it?  In Java, my understanding is it is preferable to return the interface, (i.e. Set), and leave the implementation, (HashSet,) as an implementation detail in the method.